backend/model/trees_mixin.rb

```diff
@@ -71,8 +71,8 @@ def quick_containers
         :type_3 => row[:sub_container_type_3],
         :indicator_3 => row[:sub_container_indicator_3],
       }
-      result[row[:archival_object_id]] = fields
-      
+      result[row[:archival_object_id]] ||= []
+      result[row[:archival_object_id]] << fields
```

Comment:
What's going on here?  Seems that we're changing the type from a hash to an array of hashes here? Was this just broken all along?